### ISSUE TITLE
[BUG] In some cases, 0.0.29+ generates no docs unless <src> argument is specified

### ISSUE THREAD

**Issue by brian428** [compodoc/compodoc#63]
I had been using compodoc 0.27 just fine. Tonight I updated to 0.31 and the generator fails to parse any files. Since the only thing I changed is the version of compodoc, I have to assume something changed in compodoc itself.

I'm running compodoc the same way I always have: `node_modules/.bin/compodoc -p src/client/tsconfig.json`.

With 0.27, I see a lot of output as my project was processed. Now, all I see is:

```
[00:40:44] Searching package.json file
[00:40:44] package.json file found
[00:40:44] Searching README.md file
[00:40:44] README.md file found
[00:40:44] Get dependencies data
[00:40:44] parsing        : C:/Developer/myproject/gulpfile.ts
[00:40:45] Prepare modules
[00:40:45] Prepare components
[00:40:45] Process pages
[00:40:45] Process page   : index
[00:40:45] Process page   : overview
[00:40:45] Process page   : modules
[00:40:45] Copy main resources
[00:40:46] Process main graph
[00:40:49] Documentation generated in ./documentation/ in 4.954 seconds using gitbook theme
```

And serving up the resulting docs reveals an index.html with no project docs. If I revert back to 0.27, everything works again.

So, was there some crucial change between 0.27 and 0.31 that I'm not aware of? Or is this a bug (either with compodoc or related to its dependency on TypeScript 2.1.5)?

Thanks.

**Comment by brian428** [comment]
I figured it out. Seems to be related to the new <src> argument. When I try it on 0.0.31 with `node_modules/.bin/compodoc ./src/client -p src/client/tsconfig.json`, it works again. I'll update the issue title a bit in case anyone else runs into this.

**Comment by vogloblinsky** [comment]
Fixed in v0.0.32

**Comment by brian428** [comment]
Ah, nice. I thought I had just missed the need to use src in the docs, didn't realize this was actually a bug! Thanks.
